src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/job/GenotypedVcfJobParametersValidator.java

```diff
@@ -53,11 +52,10 @@ private CompositeJobParametersValidator compositeJobParametersValidator(JobParam
 
         Boolean skipAnnotation = Boolean.valueOf(jobParameters.getString(JobParametersNames.ANNOTATION_SKIP));
         if (!skipAnnotation) {
-            // the next validator is required only in some jobs that use the annotation steps, the steps don't include it
-            jobParametersValidators.add(new InputStudyIdValidator());
+            boolean studyIdRequired = true;
 
             jobParametersValidators.add(new VepInputGeneratorStepParametersValidator());
-            jobParametersValidators.add(new VepAnnotationGeneratorStepParametersValidator());
+            jobParametersValidators.add(new VepAnnotationGeneratorStepParametersValidator(studyIdRequired));
```

Comment:
but in fact, the annot load step validator does require that logic as well, and do it as well on the vcf_id parameter, so I'll add those changes